var/spack/repos/builtin/packages/xtensor/package.py

```diff
@@ -27,11 +28,13 @@ class Xtensor(CMakePackage):
             description='Enable TBB parallelization')
 
     depends_on('xtl', when='@develop')
+    depends_on('xtl@0.7.2:0.7.99', when='@0.23.4')
```

Comment:
This looks like it matches the following line so I think it would be clearer to augment the `when`:

```
depends_on('xtl@0.7.2:0.7.99', when='@0.23.2:0.23.4')
```

Reply:
Done